### ISSUE TITLE
linux gcc -std=c11 segfault

### ISSUE THREAD

**Issue by whoizit** [vlang/v#479]
gcc 9.1.0
```
$ cc -std=c11 -w -o vc v.c
$ ./vc -o v .
Segmentation fault
```
```
$ cc -std=gnu11 -w -o vc v.c
$ ./vc -o v .
```

**Comment by ofunc** [comment]
~/code directory has to be used (it's a temporary limitation)

**Comment by whoizit** [comment]
I do it in `~/code/v/compiler` directory. I mean for Linux gcc it should be `-std=gnu11`

**Comment by whoizit** [comment]
#486

**Comment by Ivo-Balbaert** [comment]
I can confirm this: segfault with -std=c11, compiles ok with -std=gnu11 on Ubuntu 19

**Comment by Xiphin** [comment]
Solved it: #507

**Comment by Oxylibrium** [comment]
Compiling now uses `--std=gnu11`: https://github.com/vlang/v/commit/baa4f9d0c9253463ccd29c6ac614a8d49565c01d
